andela/Ah-backend-guardians
# #162685669 User can Favorite an Article

#### What Does This PR Do?
This PR enables a user to favorite an article
#### Link To Pivotal Tracker Story
#162685669
#### How Should This Be Manually Tested

- Fetching the branch 'ft-favorite-article-162685669' onto your local machine

- Run the App

- Fetch the Article of interest by using the url: http://127.0.0.1:8000/api/articles/slug/

- Using POST method, append the word 'favorite' on to the above url to Favorite the Article: url should appear as below; http://127.0.0.1:8000/api/articles/slug/favorite/

- Using the PUT method on the same url, the Article shall be Unfavorited
#### NOTE
You do not need to pass anything in the body
#### Snapshots
<img width="570" alt="screen shot 2019-01-22 at 9 07 33 pm" src="https://user-images.githubusercontent.com/42971167/51555913-d0909580-1e89-11e9-83ea-1415593b3a08.png">
<img width="480" alt="screen shot 2019-01-22 at 9 08 44 pm" src="https://user-images.githubusercontent.com/42971167/51555962-f027be00-1e89-11e9-9d6f-dcdc5f00ca55.png">
